### PR TITLE
Replaced FixedValue with Value

### PR DESCRIPTION
**Pull request by mcardosos**

### CODE REVIEW

**Comment by mcardosos** [comment]
PTAL @jhendirxMSFT @marstr

**Comment by mcardosos** [comment]
Fixes [this Travis failure](https://travis-ci.org/Azure/azure-sdk-for-go/jobs/275144765#L667).
According to @olydis , `FixedValue` should not be used unless it is strictly necessary,

**Comment by mcardosos** [comment]
Most important effect: there are no breaking changes in the SDK.
Interesting effects:
- `whatevergroup.go` files got renamed to `whatever.go`
- Bunch of dumb validation code generated

File: src/Model/ParameterGo.cs [hunk 1]
```diff
@@ -234,6 +234,11 @@ public static IEnumerable<ParameterGo> QueryParameters(this IEnumerable<Paramete
 
         public static string Validate(this IEnumerable<ParameterGo> parameters, HttpMethod method)
         {
+            if (!parameters.Any())
+            {
+                return null;
```
Comment (marstr) [inline src/Model/ParameterGo.cs]: Nit: I think it would be more appropriate to return string.Empty here. :)

Comment (mcardosos) [reply src/Model/ParameterGo.cs]: Discussing this with Joel, this lines were not needed at all. So deleted

**Pull request merged**